recipes/glasbey/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "glasbey" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f6bd5e539c12edfa1ec939a4ea00410348eecfa0a96aa72179c30d30ae59a1e2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - numpy >=1.22
+    - numba >=0.56
+    - matplotlib-base >=3.5
+    - colorspacious
```

Comment:
It doesn't seem that any of these are build time requirements. It should be safe to remove them
```suggestion
```